frontend/pages/about/contact-us.js

```diff
@@ -0,0 +1,267 @@
+                />
+                <h3>Tiana Douroudis</h3>
+                <Link href="mailto:treasurer@unswwit.com">
+                  <a><p align="left">treasurer@unswwit.com</p></a>
+                </Link>
+              </div>
+            </div>
+
+            <hr
+              className={[styles.lineBreak, styles.solid].join(" ")}
+              width="90%"
+            />
+            <div className={styles.col3}>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/careers-icon.png"
+                  alt="careers-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Brianna Kim</h3>
+                <Link href="mailto:careers@unswwit.com">
+                  <a><p align="left">careers@unswwit.com</p></a>
+                </Link>
+              </div>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/careers-icon.png"
+                  alt="careers-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Vivian Wang</h3>
+                <Link href="mailto:careers@unswwit.com">
+                  <a><p align="left">careers@unswwit.com</p></a>
+                </Link>
+              </div>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/comps-icon.png"
+                  alt="comps-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Vivian Wong</h3>
+                <Link href="mailto:competitions@unswwit.com">
+                  <a><p align="left">competitions@unswwit.com</p></a>
+                </Link>
+              </div>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/ex-icon.png"
+                  alt="ex-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Gayathri Balaji</h3>
+                <Link href="mailto:externals@unswwit.com">
+                  <a><p align="left">externals@unswwit.com</p></a>
+                </Link>
+              </div>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/ex-icon.png"
+                  alt="ex-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Tanisha Dasmohaptra</h3>
+                <Link href="mailto:externals@unswwit.com">
+                  <a><p align="left">externals@unswwit.com</p></a>
+                </Link>
+              </div>
+              <div className={styles.profile3}>
+                <Image
+                  src="/icons/hr-icon.png"
+                  alt="hr-icon"
+                  width="50px"
+                  height="50px"
+                />
+                <h3>Emily Chang</h3>
```

Comment:
Can we make the text center aligned for mobile sizes? Currently, it is left aligned